lib/ops/opsservice/install.go

```diff
@@ -332,13 +332,17 @@ func (s *site) updateOperationState(op *ops.SiteOperation, req ops.OperationUpda
 			ops.OperationStateInstallInitiated,
 			ops.OperationStateInstallProvisioning,
 			ops.OperationStateReady,
+			ops.OperationStateInstallPrechecks,
```

Comment:
This is required to support improved re-entrancy during install/expand. Unfortunately, current design of the operation state erases the previous value (by setting the state to `OperationFailed`) which makes it impossible to make an educated decision based on the actual previous step. This might be the future improvement though.